root/static/scripts/tests/release-editor/dialogs.js

```diff
@@ -42,12 +42,13 @@ function dialogTest(name, callback) {
             useTrackLengths: true
         };
 
-        var $fixture = $('<div>').attr('id', 'fixture')
-        .appendTo('body')
-            .append(
-                $("<div>").attr("id", "add-disc-dialog").hide(),
-                $("<div>").attr("id", "track-parser-dialog").hide()
-            );
+        var $fixture = $('<div>')
+            .attr('id', 'fixture')
+            .appendTo('body')
+                .append(
+                    $("<div>").attr("id", "add-disc-dialog").hide(),
+                    $("<div>").attr("id", "track-parser-dialog").hide()
```

Comment:
Mind adding a comma-dangle here while you're at it? (this line should also end with a comma).